React/Views/RCTMaskedView.h

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#import <UIKit/UIKit.h>
```

Comment:
I guess this is a Paper view? Do we want to update this (and other paper views) too?

Reply:
Yes, we should update all these refs to use the shim api's

Reply:
There are places in which Fabric UI Components use (shared) React/Views from the current 'Paper' UI